.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,105 @@
+name: Docker Image CI
+
+on:
+  pull_request:
+    branches:
+      - main
+      - development
+  push:
+    branches:
+      - main
+      - development
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+      - name: Export release code
+        if: (success() || failure())
+        id: releasecode #version number in a more comprehensible format: 0.1.YearMonthDay in UTC
+        run: |
+          export VERSION=$(grep APP_VERSION= .env | cut -d '=' -f2)
+          export RELEASE=$VERSION.$(date --utc +%y%m%d)
+          echo "RELEASE=$RELEASE" >> $GITHUB_ENV
+          echo "##[set-output name=releasename]$RELEASE"
+      - name: Setting APP_NAME
+        run: |
+          export NAME=$(grep APP_NAME= .env | cut -d '=' -f2)
+          echo "APP_NAME=$NAME" >> $GITHUB_ENV
+      - name: Print app name
+        run: echo "APP_NAME = $APP_NAME"
+      - name: Setting APP_ENV to dev
+        run: |
+          echo "APP_ENV=dev">> $GITHUB_ENV
+          echo "APP_BUILD=dev">> $GITHUB_ENV
+          echo "set APP_ENV to $APP_ENV"
+      - name: Setting APP_ENV to prod
+        if: contains( github.ref, 'master' ) || contains( github.base_ref, 'master' )
```

Comment:
Shouldn't this be main now ? or at least not master any more

Reply:
Good point